test/unit.js

```diff
@@ -0,0 +1,273 @@
+import camelCase from 'camel-case';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ReactTestUtils from 'react-addons-test-utils';
+import reactify from '../src/index';
+import skate from 'skatejs';
+import helperElement from 'skatejs/test/lib/element';
+
+describe('react-integration;', function () {
+    var fixture;
+
+    beforeEach(function () {
+        fixture = document.createElement('div');
+        document.body.appendChild(fixture);
+    });
+
+    afterEach(function () {
+        ReactDOM.unmountComponentAtNode(fixture);
+        fixture.innerHTML = '';
+        document.body.removeChild(fixture);
+    });
+
+    var tagName;
+
+    function getWebComponent () {
+        return document.querySelector(tagName);
+    }
+
+    function makeReactComponent (webComponentConstructor) {
+        var ReactComponent = reactify(webComponentConstructor, {
+            React: React,
+            ReactDOM: ReactDOM
+        });
+
+        return ReactComponent;
+    }
+
+    function renderApp(App) {
+        App.displayName = 'App';
+        return ReactDOM.render(<App />, fixture);
+    }
+
+    describe('(React > React wrapped terminal web component);', function () {
+        describe('that has an attribute;', function () {
+            var ReactComponent;
+
+            beforeEach(function () {
+                tagName = helperElement('x-terminal-text').safe;
+                var webComponentConstructor = skate(tagName, {
+                    render: function (elem) {
+                        elem.innerHTML = '<em><input />' + elem.text + '</em>';
+                    },
+                    properties: {
+                        text: {
+                            attribute: true,
+                            set: function (elem) {
+                                skate.render(elem);
+                            }
+                        }
+                    }
+                });
+
+                ReactComponent = makeReactComponent(webComponentConstructor);
+            });
+
+            it('the web component renders correctly', function () {
+                renderApp(
+                    React.createClass({
+                        render: function () {
+                            return <ReactComponent/>;
+                        }
+                    })
+                );
+
+                expect(getWebComponent().firstChild.tagName).to.equal('EM');
+            });
+
+            it('the react component renders props through the attribute', function () {
+                const WEB_COMPONENT_TEXT = 'terminal component';
```

Comment:
I've generally been using `const` where I can in place of both `let` and `var` which makes yelling the varname a bit cumbersome and ugly (subjective?). That said, it's probably better to just do `webComponentText`.